packages/smooth_app/lib/l10n/app_bg.arb

```diff
@@ -418,49 +421,49 @@
         "description": "Product compatibility summary title"
     },
     "@Plural": {},
-    "pct_match": "{percent}% match",
+    "pct_match": "{percent}% съвпадение",
     "@pct_match": {
         "description": "This product has a x percent match with your preferences",
         "placeholders": {
             "percent": {}
         }
     },
-    "plural_ago_days": "{count,plural, =1{one day ago} other{{count} days ago}}",
+    "plural_ago_days": "{count,plural, one {} =1{преди един ден} other{преди {count} дни}}",
```

Comment:
```suggestion
    "plural_ago_days": "{count,plural, =1{преди един ден} other{преди {count} дни}}",
```

Reply:
it's also appearing in Danish and Hebrew, it's probably the old non simplified version of the string. I'm going to fix them